### ISSUE TITLE
Unable to process file command 'output' successfully

### ISSUE THREAD

**Issue by skibinstein** [koslib/helm-eks-action#48]
Hi, 

After the latest change related to GitHub outputs, the GitHub action is failing.
It deploys the helm chart correctly but fails to print the helm output. 

```
Run koslib/helm-eks-action@master
/usr/bin/docker run ....


running entrypoint command(s)
Error: Unable to process file command 'output' successfully.
Error: Invalid format 'NAME: api'
```

**Comment by koslib** [comment]
Hey @skibinstein and thanks for reporting. You can switch to the latest `1.25.x` version that works for you until a fix is out. I am working on it though and should release a fixed `1.26.x` version within the next couple of days.

**Comment by koslib** [comment]
@skibinstein I've reverted the offending PR while I'm looking for a solution. Hopefully the pipelines don't fail now (confirmed through some test pipelines of mine at least).

**Comment by riontric** [comment]
@koslib 
hi, you reverted that commit [5afb272d43675e103430081e50435f25e9eaa4a0](https://github.com/koslib/helm-eks-action/commit/5afb272d43675e103430081e50435f25e9eaa4a0), but i think so that was correct, or wasn't true?

**Comment by koslib** [comment]
Fixed by #54
